us-02/us-02.py

```diff
@@ -78,6 +82,11 @@ def csvRead(path):
         for row in csvReader:
 
             currentRow = row[0]
+            
+             print(currentRow)
+            
+            nltk_tokens = nltk.word_tokenize(currentRow)
```

Comment:
so, i think nltk_tokens will reset with each response since this is in a for loop. i think we should abstract out the sematic analysis stuff to avoid this. this is also the csvRead() function. it's purpose is only to read the csv file.